changelog.txt

```diff
@@ -1,4 +1,7 @@
 *** Changelog ***
+= 5.2.0 - 2021-04-28 =
```

Comment:
This should be `= 5.2.0 - 2021-xx-xx =`, the `xx-xx` gets replaced during the release process:
```suggestion
= 5.2.0 - 2021-xx-xx =
```

Reply:
Gotcha. I didn't know that. I'll update

Reply:
done